src/detections/filters/accordion-details/filter/wrapper.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useProps } from 'detections/filters/accordion-details/filter/use-props'
+
+export const Wrapper = ({ children }) => <Grid {...useProps()}>
+  {children}
+</Grid>
+
+Wrapper.propTypes = { children: PropTypes.element.isRequired }
```

Comment:
You sure this wrapper receives only one element as its children?

Reply:
done